sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java

```diff
@@ -252,6 +266,20 @@ protected TransactionsFilter clone() throws CloneNotSupportedException {
 		return map;
 	}
 
+	private String constructInclude() {
```

Comment:
There's `?include=Account,Merchant` in `getTransactions` method in `MenigaAPI.java`. I guess it's not needed anymore.

Reply:
Oh yes, need to remove that. I was wondering how they were getting included.